src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/tests/Toggler.test.js

```diff
@@ -1,46 +1,50 @@
 // @flow
 import React from 'react';
-import {shallow, render} from 'enzyme';
+import {fireEvent, render, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples